hackclub.com.yaml

```diff
@@ -294,10 +295,6 @@ aloha:
   - ttl: 1
     type: CNAME
     value: epic-mcnulty-0ed53c.netlify.app.
-alopb:
-  - ttl: 1
-    type: CNAME
-    value: alopb.netlify.app.
```

Comment:
I don't think this is what you want. This will prevent `alopb.hackclub.com` from pointing anywhere, including vercel. I think what you're looking for is changing the value to `cname.vercel-dns.com.`, in addition to adding the domain verification